scripts/webhooks.js

```diff
@@ -17,7 +17,7 @@ const main = async () => {
   })
 
   // Fetch triggers
-  const { data } = await client.stackClient.fetchJSON('GET', '/jobs/triggers')
+  const { data } = await client.collection('io.cozy.triggers').all()
```

Comment:
Even better: `await client.queryAll(Q('io.cozy.triggers'))`. We try not using `.collection` when it's possible to use CozyClient methods directly